packages/gravity-ui-web/src/sass/05-components/03-organisms/_list-cards-basic.scss

```diff
@@ -1,5 +1,5 @@
 .grav-c-list-cards-basic {
-  margin: 0 -$grav-sp-l;
+  margin: 0;
```

Comment:
As per our conventions, we should avoid setting explicit margins "outside" our components as this intereferes with the global `* + *` rule we have to add a margin between successive, stacked blocks.

I realise now that this was happening even before this change. I missed it in the last review. Sorry.